### DISCUSSION TITLE
Breakout room 1

### DISCUSSION THREAD

**Discussion by benmcgeachie** [IBAHCM/RPiR#141] [Archive]
Hello, 

the people in Breakout room 1 were

Alexa Roditi - @AlexaRoditi
Bakari Mtili - @2500251M,  and 
Ben Mcgeachie - @benmcgeachie

**Comment by richardreeve** [comment]
Thanks @benmcgeachie - I'm moving this from Announcements to Registration. Do be careful where you put things though!
